qt/python/mantidqt/mantidqt/plotting/test/test_sample_shape.py

```diff
@@ -116,42 +116,42 @@ def tearDown(self) -> None:
     def test_CSG_merged_shape_is_valid(self):
         workspace = setup_workspace_shape_from_CSG_merged()
         self.assertTrue(workspace.sample().getShape())
-        self.assertTrue(sample_shape.get_valid_sample_shape_from_workspace(workspace))
+        self.assertTrue(sample_shape.get_valid_sample_mesh_from_workspace(workspace) is not None)
```

Comment:
You could use `self.assertIsNotNone()` for these if you like.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone